# Automatically Generated in 2017-06-12 22:33:47.452972
# MESSAGE: ODOMETER
# Description:Measurement of Odometry Sensor
uint8 ID = 140
uint8 sysid
uint8 compid
uint32 timestamp
float32 time_delta
float32 xdist_delta
float32 ydist_delta
float32 zdist_delta
float32 xdist_abs
float32 ydist_abs
float32 zdist_abs
float32 xvelocity
float32 yvelocity
float32 zvelocity
int16 quality
